package.json

```diff
@@ -12,12 +12,13 @@
     "lint": "gts check",
     "lint:fix": "gts fix",
     "start": "ts-node index.ts",
-    "test:build": "npm run compile:spectypes && npm run compile:ts-runtime-checks && npm run compile:typebox && npm run compile:typia && npm run compile:deepkit && npm run compile:ts-auto-guard && npm run compile:type-predicate-generator && tsc --noEmit",
-    "test": "npm run compile:spectypes && npm run compile:ts-runtime-checks && npm run compile:typebox && npm run compile:typia && npm run compile:deepkit && npm run compile:ts-auto-guard && npm run compile:type-predicate-generator && vitest run",
+    "test:build": "npm run compile:spectypes && npm run compile:ts-runtime-checks && npm run compile:typebox && npm run compile:typia && npm run compile:deepkit && npm run compile:ts-auto-guard && npm run compile:type-predicate-generator && npm run compile:paseri && tsc --noEmit",
+    "test": "npm run compile:spectypes && npm run compile:ts-runtime-checks && npm run compile:typebox && npm run compile:typia && npm run compile:deepkit && npm run compile:ts-auto-guard && npm run compile:type-predicate-generator && npm run compile:paseri && vitest run",
     "docs:serve": "serve docs",
     "docs:watch": "tsc --project tsconfig.docs.json --watch --preserveWatchOutput & serve docs",
     "docs:build": "tsc --project tsconfig.docs.json",
     "compile:deepkit": "deepkit-type-install && rimraf cases/deepkit/build && tsc -p cases/deepkit/tsconfig.json",
+    "compile:paseri": "rimraf cases/paseri/build && esbuild cases/paseri/src/index.ts --bundle --minify --platform=node --outdir=cases/paseri/build && tsc --emitDeclarationOnly cases/paseri/src/index.ts --outDir cases/paseri/build",
```

Comment:
Why is the esbuild buildstep necessary? is it because paseri targets deno and won't run on node without compiling?

Reply:
Nothing to do with Deno. Paseri is ESM-only, while this library is CJS. You can’t mix import types, so we need to recompile it to CJS to match.

Reply:
Ah yeah of course. Another maintenance task incoming: switch to ESM :grin:

Reply:
See my previous PR where I did just that 😂